baybe/utils/scaling.py

```diff
@@ -0,0 +1,37 @@
+"""Scaling utilities."""
+
+from __future__ import annotations
+
+from enum import Enum
+from typing import TYPE_CHECKING, Literal, TypeAlias
+
+if TYPE_CHECKING:
+    from sklearn.base import BaseEstimator, TransformerMixin
+
+    Scaler: TypeAlias = BaseEstimator | TransformerMixin
+
+
+class ScalingMethod(Enum):
```

Comment:
why go via this intermediate enum and not make `_get_parameter_scaling` directly return a `ScalerProtocol`? If we ever have scalers that have additioanl args and different surrogates want to set the differently that flexibility is easily included there. Also is there a need for having the enum that I dont see?

Reply:
The reason was also partly because I didn't like that our methods return sklearn objects. But I like the idea of wrapping this into a protocol a lot (I think by now you know that I'm a fan :D) so happily changed it. Thanks for the great suggestions, don't know why it didn't come to my mind